crates/sui-node/src/lib.rs

```diff
@@ -1697,6 +1698,12 @@ impl SuiNode {
         }
     }
 
+    async fn shutdown(self: Arc<Self>) {
```

Comment:
Does it have to be Arc<Self> or can it be just &Self? Number of Arcs can be sensitive at the time of shutting down something, not sure.

Reply:
Yeah this worked.